docs/models/successful_responses.md

```
@@ -580,6 +580,8 @@ Custom profile customization for prompts and validation.
 |-------|------|-------------|
 | path | string | Path to Python modules containing custom profile. |
 | prompts | object | Dictionary containing map of system prompts |
+| validation | string | Read-only. Loaded from the custom profile Python module (PROFILE_CONFIG["system_prompts"]["validation"]), not from lightspeed-stack.yaml. Used when a question_validity shield omits model_prompt. |
```

markdownlint-cli2 (0.23.2)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: ""validation""

(MD052, reference-links-images)